cluster/src/main/java/com/hartwig/pipeline/tertiary/cuppa/Cuppa.java

```diff
@@ -106,27 +122,97 @@ public VirtualMachineJobDefinition vmDefinition(final BashStartupScript bash, fi
     @Override
     public CuppaOutput output(final SomaticRunMetadata metadata, final PipelineStatus jobStatus, final RuntimeBucket bucket,
             final ResultsDirectory resultsDirectory) {
+        final String conclusionTxt = conclusionTsv(metadata);
+        final String cuppaChart = cuppaChart(metadata);
+        final String resultsCsv = resultCsv(metadata);
+        final String featurePlot = featurePlot(metadata);
         return CuppaOutput.builder()
                 .status(jobStatus)
+                .conclusionTxt(GoogleStorageLocation.of(bucket.name(), resultsDirectory.path(conclusionTxt)))
+                .chartPng(GoogleStorageLocation.of(bucket.name(), resultsDirectory.path(cuppaChart)))
+                .featurePlot(GoogleStorageLocation.of(bucket.name(), resultsDirectory.path(featurePlot)))
+                .resultCsv(GoogleStorageLocation.of(bucket.name(), resultsDirectory.path(resultsCsv)))
                 .addFailedLogLocations(GoogleStorageLocation.of(bucket.name(), RunLogComponent.LOG_FILE))
                 .addReportComponents(new EntireOutputComponent(bucket, Folder.root(), namespace(), resultsDirectory))
                 .addDatatypes(new AddDatatype(DataType.CUPPA_CHART,
                         metadata.barcode(),
-                        new ArchivePath(Folder.root(), namespace(), metadata.tumor().sampleName() + ".cuppa.chart.png")))
+                        new ArchivePath(Folder.root(), namespace(), cuppaChart)))
                 .addDatatypes(new AddDatatype(DataType.CUPPA_CONCLUSION,
                         metadata.barcode(),
-                        new ArchivePath(Folder.root(), namespace(), metadata.tumor().sampleName() + ".cuppa.conclusion.txt")))
+                        new ArchivePath(Folder.root(), namespace(), conclusionTxt)))
+                .addDatatypes(new AddDatatype(DataType.CUPPA_RESULTS,
+                        metadata.barcode(),
+                        new ArchivePath(Folder.root(), namespace(), resultsCsv)))
+                .addDatatypes(new AddDatatype(DataType.CUPPA_FEATURE_PLOT,
+                        metadata.barcode(),
+                        new ArchivePath(Folder.root(), namespace(), featurePlot)))
                 .build();
     }
 
+    @NotNull
+    protected String featurePlot(final SomaticRunMetadata metadata) {
+        return metadata.tumor().sampleName() + CUP_REPORT_SUMMARY_PNG;
+    }
+
+    @NotNull
+    protected String resultCsv(final SomaticRunMetadata metadata) {
+        return metadata.tumor().sampleName() + CUP_DATA_CSV;
+    }
```

Comment:
Can we make the constants private/protected/package-private? And make the methods match the constant names?